xskillscore/tests/test_metric_results_accurate.py

```diff
@@ -31,6 +31,10 @@
     (mape, mean_absolute_percentage_error),
 ]
 
+xs_skl_metrics_with_zeros = [
+    (mape, mean_absolute_percentage_error),
+]
+
 xs_scipy_metrics = [
```

Comment:
thats not needed anymore, right?

Reply:
yeah it used in the loop over metrics we want to test again when they encounter 0. MAPE is the only relevant one at the moment but as you mentioned we could add in the epsilon in other metrics in the future and expand this list.